src/Security/Authentication/Core/src/AuthenticationHandler.cs

```diff
@@ -156,7 +156,7 @@ public async Task InitializeAsync(AuthenticationScheme scheme, HttpContext conte
 
         TimeProvider = Options.TimeProvider ?? TimeProvider.System;
 #pragma warning disable CS0618 // Type or member is obsolete
-        Clock = new SystemClock(TimeProvider);
+        Clock = TimeProvider == TimeProvider.System ? SystemClock.Default : new SystemClock(TimeProvider);
```

Comment:
```suggestion
        Clock = (TimeProvider == TimeProvider.System) ? SystemClock.Default : new SystemClock(TimeProvider);
```
?

Reply:
Is using parens here the predominant style in this repo?

Reply:
I'm not at a computer so I can't check 😊 personally I find it more readable (I at first read the == as =) and in other cases it can possibly avoid precedence errors?

Reply:
For reference, a regex search in aspnetcore/src for `\(\w+ == \w+\) \? ` yields 21 matches whereas a regex search for `\w+ == \w+ \? ` yields 201 matches.

Someone who spends more time than me in dotnet/aspnetcore tell me what they want this to be and I'll make it so.

Reply:
For simple conditions like this the parens don't add much.

Reply:
We usually use non-parens in this repo. At least for a single `==` check. If it was more complex like `(foo == bar || foo == baz) ? doThing ? doOtherThing;` Then I think parens starts making sense.

We do generally use `Instance` for static default instantiations of a class though.
https://github.com/search?q=repo%3Adotnet%2Faspnetcore+%22instance+%3D%22&type=code